### PR TITLE
update write_to_h5_dataset_format

### PR DESCRIPTION
**Pull request by ferchaure**
Hi Guys,  a few changes for write_to_h5_dataset_format:

- new argument _single\_axis_ to save vectors with only one dimension (mainly for compatibility with combinato).
- changed default value of _chunk\_memory_ to an int (I guess this was a little bug).

let me know what you think

### CODE REVIEW

File: spikeinterface/core/core_tools.py [hunk 1]
```diff
@@ -347,7 +347,7 @@ def write_memory_recording(recording, dtype=None, verbose=False, **job_kwargs):
 
 
 def write_to_h5_dataset_format(recording, dataset_path, segment_index, save_path=None, file_handle=None,
-                               time_axis=0, dtype=None, chunk_size=None, chunk_memory='500M', verbose=False):
+                               time_axis=0, single_axis=False, dtype=None, chunk_size=None, chunk_memory=500, verbose=False):
```
Comment (samuelgarcia) [inline spikeinterface/core/core_tools.py:350]: chunk_memory= str  ('500M' or '1G')
chunk_size = integer

**Comment by samuelgarcia** [comment]
Hi Ferdando,
this looks good.
except the chunk_memory that is now a str.

**Comment by samuelgarcia** [comment]
chunk_size/chunk_memory/chunk_memory have change in this new release.
It is now more flexible and support str indeed!!
Have a look to this. 
https://github.com/SpikeInterface/spikeinterface/blob/master/spikeinterface/core/job_tools.py#L70

**Comment by ferchaure** [comment]
then I will have to import the function _mem_to_int from core/job_tools.py

**Comment by samuelgarcia** [comment]
Hi.
in this function we already have 
```
chunk_size = ensure_chunk_size(recording, chunk_size=chunk_size, chunk_memory=chunk_memory, n_jobs=1)`
```

But there is a bug theses lines have to be removed. It is a mistake from me.

```
    if chunk_size is not None:
        chunk_size = int(chunk_size)
    elif chunk_memory is not None:
        n_bytes = np.dtype(recording.get_dtype()).itemsize
        max_size = int(chunk_memory * 1e6)  # set Mb per chunk
        chunk_size = max_size // (num_channels * n_bytes)
```

**Comment by ferchaure** [comment]
now?

**Comment by samuelgarcia** [comment]
Perfect.
I run tests and I will merge.

**Comment by samuelgarcia** [comment]
Do you want to also change the cambinato writing also ?https://github.com/SpikeInterface/spikeinterface/blob/master/spikeinterface/sorters/combinato/combinato.py

**Comment by ferchaure** [comment]
yes, I already did the new code for combinato, I just doing one pull request at a time.

I will check the wave_clus sorter as well.

**Review by samuelgarcia — APPROVED** [review]

**Comment by samuelgarcia** [comment]
Ok merged.

**Pull request merged**